generators/server/templates/src/main/java/package/config/DatabaseConfiguration_couchbase.java.ejs

```diff
@@ -179,7 +179,7 @@ public class DatabaseConfiguration extends AbstractCouchbaseConfiguration {
     @Bean
     @Profile({JHipsterConstants.SPRING_PROFILE_TEST, JHipsterConstants.SPRING_PROFILE_E2E})
     public Couchmove waitForIndexes(Couchmove couchmove) {
-        couchmove.waitForN1qlIndexes(getScopeName(), Duration.ofSeconds(60));
+        couchmove.waitForN1qlIndexes(getScopeName(), Duration.ofMinutes(15));
```

Comment:
15min looks too high

Reply:
The only way we've found with @tchlyah to have it pass with github ci

Reply:
Still, backend tests takes 3min.
15min is the entire workflow length many times.
Would be better to reduce to 3min at most.